internal/manager/manager_tasks.go

```diff
@@ -393,7 +395,7 @@ func (s *Manager) StashBoxBatchPerformerTag(ctx context.Context, box *models.Sta
 	j := job.MakeJobExec(func(ctx context.Context, progress *job.Progress) error {
 		logger.Infof("Initiating stash-box batch performer tag")
 
-		var tasks []StashBoxBatchTagTask
+		var tasks []Task
 
 		// The gocritic linter wants to turn this ifElseChain into a switch.
 		// however, such a switch would contain quite large blocks for each section
```

Comment:
Since we're modifying this code already, I think it's worth resolving this golint issue.

Use a `switch` block, and extract each case into a private method that will return the tasks.